atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/async/CqlClientImpl.java

```diff
@@ -97,7 +97,7 @@ public static CqlClient create(
                 tuningConfig.preparedStatementCacheSize());
     }
 
-    private static CqlClient create(
+    static CqlClient create(
```

Comment:
nit: There's an annotation `@VisibleForTesting` you can use for this kind of broadening that's only for tests

Reply:
This is wrong refactoring but I added it to the constructor.